client_end_user/shared/src/commonMain/kotlin/resources/strings/en/English.kt

```diff
@@ -73,4 +73,6 @@ data class English(
     override val thisWeek: String = "This Week",
     override val tryAgain: String = "Try again",
     override val trackYourOrder: String = "Track Your Order",
-) : IStringResources
\ No newline at end of file
+) : IStringResources {
+    override val history: String = "History"
```

Comment:
Please make history val an override constructor val. Like trackYourOrder

Reply:
You are right , thanks for your notice ❤